libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -257,12 +268,28 @@ export class DropdownComponent
     }
   }
 
+  private setCardAlignment(entry) {
+    // If card alignment is left, and the entry is cut off to the right by ${entry.boundingClientRect.right - entry.intersectionRect.right}px
+    // it is set to align to end in stead, and vice versa for right-aligned card
+    if (this.cardAlign === cardAlignment.left) {
+      if (entry.boundingClientRect.right > entry.rootBounds.right) {
+        this._horizontal = 'end';
+        this.cardAlign = cardAlignment.right;
```

Comment:
Maybe these to lines are hinting at wanting to be the same thing/concept as mentioned in the comment regarding `cardAlign` naming? 🤔

Reply:
Refactored